### PR TITLE
POJOs - 3

### PR DESCRIPTION
**Pull request by st-pasha**
Implementing GBM raw models

---
This change is [Reviewable](https://reviewable.io/reviews/h2oai/h2o-3/162)

### CODE REVIEW

File: h2o-genmodel/src/main/java/hex/genmodel/utils/Distribution.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+import static hex.genmodel.utils.Distribution.Family.bernoulli;
+import static hex.genmodel.utils.Distribution.Family.modified_huber;
+import static hex.genmodel.utils.Distribution.Family.multinomial;
+import static java.lang.Math.exp;
+
+/**
+ * Copy of `hex.Distribution.Family`.
+ */
+public class Distribution {
```
Comment (arnocandel) [inline h2o-genmodel/src/main/java/hex/genmodel/utils/Distribution.java]: Can we use one "golden" class here as well?

Comment (st-pasha) [inline h2o-genmodel/src/main/java/hex/genmodel/utils/Distribution.java]: Hmm, hex.Distribution is Iced, and we can't have anything Iced in genmodel...

Comment (mmalohlava) [inline h2o-genmodel/src/main/java/hex/genmodel/utils/Distribution.java]: Can we replace Distribution with `enum` (then it will be Serializable) ?

**Comment by arnocandel** [comment]
yes, great.

File: h2o-genmodel/src/main/java/hex/genmodel/utils/Distribution.java (hunk 1 above)
Comment (st-pasha) [inline h2o-genmodel/src/main/java/hex/genmodel/utils/Distribution.java]: Alright check the latest commit -- now DistributionFamily is in hex.genmodel.utils, and all link functions as well.

**Comment by st-pasha** [comment]
All tests have passed -- merging

**Pull request merged**